vpd-tool/include/vpd_tool.hpp

```diff
@@ -39,5 +61,20 @@ class VpdTool
                     const std::string& i_recordName,
                     const std::string& i_keywordName, const bool i_onHardware,
                     const std::string& i_fileToSave = {});
+
+    /**
+     * @brief Dump the given inventory object in JSON format to console.
+     *
+     * For a given Object Path of a FRU, this API dumps the following properties
```

Comment:
object path

Reply:
Done

Reply:
> on failure empty JSON should be sent ?

Why? On error scenario, we are throwing an exception and this is mentioned by the doxygen. What advantage does returning an empty JSON give?